docs/syntax-reference.md

```diff
@@ -679,11 +678,22 @@ conventions, which are slightly different to the syntax of the SET statement abo
 $ cat ksql.properties
 auto.offset.reset=earliest
 
-# Start KSQL in standalone mode with the custom properties above
-$ ksql-cli local --properties-file ./ksql.properties
+# Start a KSQL cli session with the custom properties above
+$ ksql http://localhost:8080 --properties-file ./ksql.properties
+```
+
+Finally, you can provide these configurations to the KSQL server in its properties file. When configuring Kafka Streams
+and Kafka via the server properties file, you must prefix the config names with `ksql.core.streams`. The KSQL server will
```

Comment:
What do you mean by "Kafka" in "when configuring Kafka streams and _Kafka_"? Can you provide Kafka server properties here? I think not. You probably mean the Kafka client properties. If so, it would be good to clarify.

Reply:
These are indeed kafka consumer and producer properties that will be used in generated streams apps.